2019-04-14 16:25:29,069 : INFO : Machine : Mission Pinball Framework Core Engine v0.53.0-dev.16
2019-04-14 16:25:29,070 : INFO : Machine : Command line arguments: {'no_load_cache': False, 'create_config_cache': True, 'bcp': True, 'configfile': ['config.yaml'], 'mpfconfigfile': 'c:\\users\\wilder\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\mpf\\mpfconfig.yaml', 'force_assets_load': False, 'jsonlogging': False, 'logfile': 'logs\\2019-04-14-16-25-29-mpf-RedTunic.log', 'pause': False, 'production': False, 'text_ui': True, 'loglevel': 10, 'consoleloglevel': 20, 'force_platform': None, 'syslog_address': None, 'mc_file_name': None, 'no_sound': False}
2019-04-14 16:25:29,070 : INFO : Machine : MPF path: c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf
2019-04-14 16:25:29,070 : INFO : Machine : Machine path: C:\Users\Wilder\zelda
2019-04-14 16:25:29,070 : INFO : Machine : Platform: win32
2019-04-14 16:25:29,070 : INFO : Machine : Python executable location: c:\users\wilder\appdata\local\programs\python\python36\python.exe
2019-04-14 16:25:29,070 : INFO : Machine : Python version: 3.6.4 (64-bit)
2019-04-14 16:25:29,070 : INFO : Machine : Machine config file #1: config.yaml
2019-04-14 16:25:29,072 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\9bf880992bc5b85aa1bd2cc37b214e56.mpf_cache
2019-04-14 16:25:29,076 : DEBUG : asyncio : Using selector: SelectSelector
2019-04-14 16:25:29,077 : INFO : Machine : Initialise MPF.
2019-04-14 16:25:29,121 : DEBUG : FAST : Configuring FAST hardware.
2019-04-14 16:25:29,324 : INFO : FAST : Connecting to \\.\COM11 at 921600bps
2019-04-14 16:25:29,357 : INFO : FAST : Connected! Processor: NET, Board Type: FP-CPU-002-2, Firmware: 01.02
2019-04-14 16:25:29,358 : INFO : EventManager : Event: ======'machine_var_fast_net_firmware'====== Args={'value': '01.02', 'prev_value': None, 'change': True}
2019-04-14 16:25:29,358 : INFO : EventManager : Event: ======'machine_var_fast_net_model'====== Args={'value': 'FP-CPU-002-2', 'prev_value': None, 'change': True}
2019-04-14 16:25:29,358 : WARNING : FAST : Not resetting FAST NET because firmware is currently broken.
2019-04-14 16:25:29,421 : INFO : FAST : Connecting to \\.\COM12 at 921600bps
2019-04-14 16:25:29,469 : INFO : FAST : Connected! Processor: RGB, Board Type: FP-CPU-002-2, Firmware: 01.00
2019-04-14 16:25:29,469 : INFO : EventManager : Event: ======'machine_var_fast_rgb_firmware'====== Args={'value': '01.00', 'prev_value': None, 'change': True}
2019-04-14 16:25:29,469 : INFO : EventManager : Event: ======'machine_var_fast_rgb_model'====== Args={'value': 'FP-CPU-002-2', 'prev_value': None, 'change': True}
2019-04-14 16:25:29,470 : INFO : EventManager : Event: ======'machine_var_credits_string'====== Args={'value': 'FREE PLAY', 'prev_value': None, 'change': True}
2019-04-14 16:25:29,527 : DEBUG : YAML File Interface : Loading file: C:\Users\Wilder\zelda\data/machine_vars.yaml
2019-04-14 16:25:29,531 : INFO : EventManager : Event: ======'machine_var_player1_score'====== Args={'value': 306, 'prev_value': None, 'change': True}
2019-04-14 16:25:29,531 : INFO : EventManager : Event: ======'machine_var_mpf_version'====== Args={'value': 'MPF v0.53.0-dev.16', 'prev_value': None, 'change': True}
2019-04-14 16:25:29,531 : INFO : EventManager : Event: ======'machine_var_mpf_extended_version'====== Args={'value': 'MPF v0.53.0-dev.16, Config version:5, Show version: 5, BCP version:1.1', 'prev_value': None, 'change': True}
2019-04-14 16:25:29,531 : INFO : EventManager : Event: ======'machine_var_python_version'====== Args={'value': '3.6.4', 'prev_value': None, 'change': True}
2019-04-14 16:25:29,532 : INFO : EventManager : Event: ======'machine_var_platform'====== Args={'value': 'Windows-7-6.1.7601-SP1', 'prev_value': None, 'change': True}
2019-04-14 16:25:29,532 : INFO : EventManager : Event: ======'machine_var_platform_system'====== Args={'value': 'Windows', 'prev_value': None, 'change': True}
2019-04-14 16:25:29,532 : INFO : EventManager : Event: ======'machine_var_platform_release'====== Args={'value': '7', 'prev_value': None, 'change': True}
2019-04-14 16:25:29,532 : INFO : EventManager : Event: ======'machine_var_platform_version'====== Args={'value': '6.1.7601', 'prev_value': None, 'change': True}
2019-04-14 16:25:29,532 : INFO : EventManager : Event: ======'machine_var_platform_machine'====== Args={'value': 'AMD64', 'prev_value': None, 'change': True}
2019-04-14 16:25:29,532 : INFO : EventManager : Event: ======'init_phase_1'====== Args={}
2019-04-14 16:25:29,571 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\6fc1a42fdd0d487285862f5b943c745e.mpf_cache
2019-04-14 16:25:29,574 : DEBUG : ModeController : Loaded mode rupee_finder
2019-04-14 16:25:29,574 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\225554da95d8871f7382aa94a6d6b3eb.mpf_cache
2019-04-14 16:25:29,577 : DEBUG : ModeController : Loaded mode lost_woods_starter
2019-04-14 16:25:29,578 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\7b0dcc699efc7732bf9d3e57ee202517.mpf_cache
2019-04-14 16:25:29,581 : DEBUG : ModeController : Loaded mode 2ballescape
2019-04-14 16:25:29,582 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\c3fd2c292a6c0d3463ad9b4555730ad5.mpf_cache
2019-04-14 16:25:29,586 : DEBUG : ModeController : Loaded mode item_tracker
2019-04-14 16:25:29,586 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\6eb8b6e4728e10ce756703e43167d298.mpf_cache
2019-04-14 16:25:29,590 : DEBUG : ModeController : Loaded mode rupee2x_finder
2019-04-14 16:25:29,590 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\86e0d165ea95889c95b56598c3ebe263.mpf_cache
2019-04-14 16:25:29,594 : DEBUG : ModeController : Loaded mode music_player
2019-04-14 16:25:29,594 : WARNING : ConfigProcessor : Config file in cache changed: C:\Users\Wilder\zelda\modes\darkworld\config\darkworld.yaml
2019-04-14 16:25:29,594 : INFO : ConfigProcessor : Loading config from file C:\Users\Wilder\zelda\modes\darkworld\config\darkworld.yaml.
2019-04-14 16:25:29,595 : DEBUG : YAML File Interface : Loading file: C:\Users\Wilder\zelda\modes\darkworld\config\darkworld.yaml
2019-04-14 16:25:29,892 : INFO : ConfigProcessor : Loading config: C:\Users\Wilder\zelda\modes\darkworld\config\darkworld.yaml
2019-04-14 16:25:29,919 : INFO : ConfigProcessor : Config file cache created: C:\Users\Wilder\AppData\Local\Temp\6cbeeb809d9de6fbbf2951c4e6867473.mpf_cache
2019-04-14 16:25:29,922 : DEBUG : ModeController : Loaded mode darkworld
2019-04-14 16:25:29,923 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\aecceeb5afbfdc7257044e8417ebcf98.mpf_cache
2019-04-14 16:25:29,928 : DEBUG : ModeController : Loaded mode attract
2019-04-14 16:25:29,928 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\1f8ff34a8859f8c62d77912f33e29b9c.mpf_cache
2019-04-14 16:25:29,932 : DEBUG : ModeController : Loaded mode 2ballqualifier
2019-04-14 16:25:29,932 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\c35ad2e595070cac0b6f696d6f7f6af6.mpf_cache
2019-04-14 16:25:29,936 : DEBUG : ModeController : Loaded mode ball_save
2019-04-14 16:25:29,936 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\58a9cd5447353eacedab412f1c10b0ab.mpf_cache
2019-04-14 16:25:29,940 : DEBUG : ModeController : Loaded mode lost_woods
2019-04-14 16:25:29,940 : WARNING : ConfigProcessor : Config file in cache changed: C:\Users\Wilder\zelda\modes\slide_tracker\config\slide_tracker.yaml
2019-04-14 16:25:29,940 : INFO : ConfigProcessor : Loading config from file C:\Users\Wilder\zelda\modes\slide_tracker\config\slide_tracker.yaml.
2019-04-14 16:25:29,940 : DEBUG : YAML File Interface : Loading file: C:\Users\Wilder\zelda\modes\slide_tracker\config\slide_tracker.yaml
2019-04-14 16:25:29,941 : ERROR : asyncio : Exception in callback EventManager._async_handler_done(<QueuedEvent>)(<Task finishe...acker.yaml',)>) at c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\events.py:96
handle: <Handle EventManager._async_handler_done(<QueuedEvent>)(<Task finishe...acker.yaml',)>) at c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\events.py:96>
Traceback (most recent call last):
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\file_interfaces\yaml_interface.py", line 197, in load
    expected_version_str, version_str, filename))
AssertionError: Version mismatch. Expected: #config_version=5 Actual: -#config_version=5 Files: C:\Users\Wilder\zelda\modes\slide_tracker\config\slide_tracker.yaml

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\asyncio\events.py", line 145, in _run
    self._callback(*self._args)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\events.py", line 99, in _async_handler_done
    future.result()
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\mode_controller.py", line 117, in load_modes
    self.machine.modes[mode] = self._load_mode(mode)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\mode_controller.py", line 266, in _load_mode
    config = self._load_mode_config(mode_string)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\mode_controller.py", line 184, in _load_mode_config
    store_to_cache=self.machine.options['create_config_cache'])
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\config_processor.py", line 112, in load_config_files_with_cache
    ignore_unknown_sections)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\config_processor.py", line 132, in _load_config_file_and_return_loaded_files
    config = FileManager.load(filename, expected_version_str, True)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\file_manager.py", line 99, in load
    return interface.load(file, verify_version, halt_on_error)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\file_interfaces\yaml_interface.py", line 212, in load
    raise ValueError(msg)
ValueError: Error found in file C:\Users\Wilder\zelda\modes\slide_tracker\config\slide_tracker.yaml: Version mismatch. Expected: #config_version=5 Actual: -#config_version=5 Files: C:\Users\Wilder\zelda\modes\slide_tracker\config\slide_tracker.yaml
2019-04-14 16:25:29,944 : INFO : Machine : Starting the main run loop.
2019-04-14 16:25:29,944 : INFO : Machine : Shutting down...
2019-04-14 16:25:29,944 : INFO : EventManager : Event: ======'shutdown'====== Args={}
2019-04-14 16:25:29,947 : ERROR : FAST : Stop called on serial connection \\.\COM12
2019-04-14 16:25:29,947 : ERROR : FAST : Stop called on serial connection \\.\COM11
2019-04-14 16:25:30,256 : ERROR : root : Error found in file C:\Users\Wilder\zelda\modes\slide_tracker\config\slide_tracker.yaml: Version mismatch. Expected: #config_version=5 Actual: -#config_version=5 Files: C:\Users\Wilder\zelda\modes\slide_tracker\config\slide_tracker.yaml
Traceback (most recent call last):
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\file_interfaces\yaml_interface.py", line 197, in load
    expected_version_str, version_str, filename))
AssertionError: Version mismatch. Expected: #config_version=5 Actual: -#config_version=5 Files: C:\Users\Wilder\zelda\modes\slide_tracker\config\slide_tracker.yaml

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\commands\game.py", line 220, in __init__
    MachineController(mpf_path, machine_path, vars(self.args)).run()
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\machine.py", line 716, in run
    self._run_loop()
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\machine.py", line 776, in _run_loop
    raise self._exception['exception']
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\asyncio\events.py", line 145, in _run
    self._callback(*self._args)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\events.py", line 99, in _async_handler_done
    future.result()
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\mode_controller.py", line 117, in load_modes
    self.machine.modes[mode] = self._load_mode(mode)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\mode_controller.py", line 266, in _load_mode
    config = self._load_mode_config(mode_string)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\mode_controller.py", line 184, in _load_mode_config
    store_to_cache=self.machine.options['create_config_cache'])
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\config_processor.py", line 112, in load_config_files_with_cache
    ignore_unknown_sections)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\config_processor.py", line 132, in _load_config_file_and_return_loaded_files
    config = FileManager.load(filename, expected_version_str, True)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\file_manager.py", line 99, in load
    return interface.load(file, verify_version, halt_on_error)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\file_interfaces\yaml_interface.py", line 212, in load
    raise ValueError(msg)
ValueError: Error found in file C:\Users\Wilder\zelda\modes\slide_tracker\config\slide_tracker.yaml: Version mismatch. Expected: #config_version=5 Actual: -#config_version=5 Files: C:\Users\Wilder\zelda\modes\slide_tracker\config\slide_tracker.yaml
